### ISSUE TITLE
ArgumentError happens when 'on_shutdown' is called

### ISSUE THREAD

**Issue by tunepolo** [dariocravero/padrino-websockets#7]
By af0c7ee277239cb1f487949de4378250ed17b415 commit, parent 'on_shutdown'(=base-event-manager.rb) invocation is added.

[lib/padrino-websockets/faye/event-manager.rb]

```ruby
def on_shutdown(event)
  @pinger.cancel if @pinger
  super
end
```

[lib/padrino-websockets/base-event-manager.rb]

```ruby
def on_shutdown
  logger.debug "Disconnecting user: #{@user} from channel: #{@channel}."
  @@connections[@channel].delete(@user)
end
```

Ruby's 'super' passes the same parameters when parameters are omitted, so ArgumentError happens.

**Comment by janus112** [comment]
@dariocravero Yes, I dont know why I did catch that one when testing, I must have tested the wrong version, sorry.
